test/src/testplane.js

```diff
@@ -49,10 +48,12 @@ describe("hermione", () => {
         sandbox.stub(RunnerStats, "create");
 
         initReporters = sandbox.stub().resolves();
+        signalHandler = new AsyncEmitter();
 
-        Hermione = proxyquire("src/hermione", {
+        Testplane = proxyquire("src/testplane", {
             "./reporters": { initReporters },
-        }).Hermione;
+            "./signal-handler": signalHandler,
+        }).Testplane;
```

Comment:
For some reason, after renaming, `signalHandler` was interfering from other files, so one tests was failing, when launched with all other test files.